meson.build

```diff
@@ -38,7 +38,8 @@ glib_dep = dependency('glib-2.0')
 gee_dep = dependency('gee-0.8')
 gio_dep = dependency('gio-2.0')
 gtk_dep = dependency('gtk+-3.0')
-granite_dep = dependency('granite', version: '>=6.1.0')
+#granite_dep = dependency('granite', version: '>=6.1.0')
+granite_dep = dependency('granite-7', version: '>= 6.1', required: true)
```

Comment:
You definitely don't want to change this because it will fail to build :) granite-7 is the GTK4 version of Granite and this project still uses GTK3